var/spack/repos/builtin/packages/palace/package.py

```diff
@@ -17,26 +17,29 @@ class Palace(CMakePackage):
     maintainers("sebastiangrimberg")
 
     version("develop", branch="main")
+    version("0.12.0", tag="v0.12.0", commit="8c192071206466638d5818048ee712e1fada386f")
     version("0.11.2", tag="v0.11.2", commit="6c3aa5f84a934a6ddd58022b2945a1bdb5fa329d")
 
-    variant("shared", default=True, description="Enables the build of shared libraries")
+    variant("shared", default=True, description="Build shared libraries")
     variant("int64", default=False, description="Use 64 bit integers")
-    variant("openmp", default=False, description="Use OpenMP")
-    variant(
-        "gslib",
-        default=True,
-        description="Build with GSLIB library for high-order field interpolation",
-    )
+    variant("openmp", default=False, description="Use OpenMP for shared-memory parallelism")
     variant(
         "superlu-dist", default=True, description="Build with SuperLU_DIST sparse direct solver"
     )
     variant("strumpack", default=False, description="Build with STRUMPACK sparse direct solver")
     variant("mumps", default=False, description="Build with MUMPS sparse direct solver")
     variant("slepc", default=True, description="Build with SLEPc eigenvalue solver")
     variant("arpack", default=False, description="Build with ARPACK eigenvalue solver")
+    variant("libxsmm", default=True, description="Build with LIBXSMM backend for libCEED")
+    variant("magma", default=True, description="Build with MAGMA backend for libCEED")
+    variant(
+        "gslib",
+        default=True,
+        description="Build with GSLIB library for high-order field interpolation",
+    )
 
     # Dependencies
-    depends_on("cmake@3.13:", type="build")
+    depends_on("cmake@3.21:", type="build")
```

Comment:
I think it's worth enforcing even for older versions even if it isn't strictly so. This package is changing quickly and I think there is little reason to enforce version backward compatibility in situations like this since CMake 3.21 is already several years old.